salesagility/SuiteCRM
# Fixed #8897 - Adding missing relationship for SurveyResponses module

Closes https://github.com/salesagility/SuiteCRM/issues/8897

The records for the related modules Contacts/Accounts, don't appear in some of the views Surveys views. Both in SurveyResponses ListView and the SurveyResponses Subpanel inside a Survey record.

## Description
Due to the issue found in the link above, we found that there are missing some relationships in the definition of the vardef between the modules SurveyResponses and Contacts/Accounts. We add the definition in the vardef definition of the SurveyResponses module.

## Motivation and Context
There is a way to define relationship between modules. This PR completes them.

## How To Test This
1. Create a Survey
2. Create a Response for that Survey
3. Add a Contact to that Response
4. Add the column Contact to the Subpanel of the Survey Response module that appear in the Survey module
5. Check that the Contact doesn't appear in the Subpanel of the Survey Response module inside the Survey Record

## Types of changes

### Final checklist
